src/runtime_src/core/pcie/linux/pcidev.h

```diff
@@ -78,22 +69,18 @@ class pci_device
 
   uint16_t domain =           INVALID_ID;
   uint16_t bus =              INVALID_ID;
-  uint16_t dev =              INVALID_ID;
+  uint16_t dev_no =           INVALID_ID;
   uint16_t func =             INVALID_ID;
-  uint16_t vendor_id =        INVALID_ID;
-  uint16_t device_id =        INVALID_ID;
   uint32_t instance =         INVALID_ID;
   std::string sysfs_name =    ""; // dir name under /sys/bus/pci/devices
   int user_bar =              0;  // BAR mapped in by tools, default is BAR0
   size_t user_bar_size =      0;
-  bool is_mgmt() const
-  {
-      return mgmt;
-  }
+  bool is_mgmt =              false;
```

Comment:
Naming convention.  Being that these variables are member variables of the class, normally an "m_" is added to them. 

Is there a reason why this isn't done in this "container" class?

Reply:
added "m_" prefix.